dask_cuda/proxify_host_file.py

```diff
@@ -181,13 +181,38 @@ def get_access_info(self) -> Tuple[int, List[Tuple[int, int, List[ProxyObject]]]
                 total_dev_mem_usage += size
             return total_dev_mem_usage, dev_buf_access
 
+    def add_external(self, obj):
+        # Notice, since `self.store` isn't modified, no lock is needed
+        found_proxies: List[ProxyObject] = []
+        proxied_id_to_proxy = {}
+        # Notice, we are excluding found objects that are already proxies
+        ret = proxify_device_objects(
+            obj, proxied_id_to_proxy, found_proxies, excl_proxies=True
+        )
+        last_access = time.time()
```

Comment:
I meant to ask about this more broadly on the proxy object effort, is the value of `time.time()` local to the current worker or also used by remote workers? If only local to the current worker, could we resort to `time.monotonic()` instead? `time.monotonic()` guarantees not going backwards, whereas `time.time()` does not, and that can be problematic in some systems.

If `time.time()` is used by remote workers, we may have a bigger problem, what if one of the workers' system time is completely messed up, what would happen then?

Reply:
Good point, we are using `last_access` to spill the _oldest_ object first so this could be an issue in principle.
Fixed.